core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -82,6 +83,8 @@ public function reset(): void
         Input::resetUnusedGet();
         InsertTags::reset();
         Registry::getInstance()->reset();
+
+        self::$nonce = '';
```

Comment:
I think this line needs to go before the if condition in line 77, otherwise it will not be reset if the framework is not initialized.

Reply:
Right, now that we're setting the nonce in the getter. Changed in 66e358f70.